keyboards/blackplum/readme.md

```diff
@@ -0,0 +1,4 @@
+
+## Blackplum 68% keyboard with RGB underglow.
+
+![Blackplum layout](https://i.imgur.com/EOUQk4J.png)
```

Comment:
Please also format this file like the template:

https://github.com/qmk/qmk_firmware/blob/master/quantum/template/avr/readme.md

Reply:
Done.